Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocTextValue.css

```diff
@@ -45,48 +45,28 @@ span.bocTextValue .content
   display: inline-block;
 }
 
-.bocTextValue.readOnly .content span[tabindex]:first-child
-{
-  /* Ensures that diacritic marks can be rendered without being cut off. */
-  padding-top: 0.2em;
-}
-
-.bocTextValue.readOnly .content span[tabindex]:last-child
-{
-  /* Ensures that diacritic marks can be rendered without being cut off. */
-  padding-bottom: 0.2em;
-}
-
 .bocTextValue.readOnly .content span[tabindex]:empty
 {
-  /* Ensures that empty readonly values are not collapsed and can therefor visualize the keyboard focus */
-  width: calc(100% - 2px);
+  /* Ensures that empty readonly values are not collapsed and can therefore visualize the keyboard focus */
+  width: 100%;
   min-height: 1.5em;
-  padding-top: 0;
-  padding-bottom: 0;
-  vertical-align: -0.2em;
+  vertical-align: middle;
```

Comment:
veritcal-align is tricky. I'm not sure if it's a) correct and b) how it works for multiline-text in readonly.
also, what happens when we remove it?

Reply:
With a bit of a workaround I managed to get rid of the vertical-align. By introducing another pseudo-class ':empty:before' and insertig a 'zero width space' character (scales with font size) there in order to ensure that the readonly value is not collapsed entirely as well as setting the width to 100% in the actual ':empty' pseudo-class, the readonly value does not collapse, is centered, and the tab-focus surrounds the entire readonly span. I'm not sure if this solution really is a good idea, though.

Reply:
okay, I'll admit, didn't check it out. Kudos for the creativity, but we should stick with the v-align. v-align isn't bad if it's necessary.

Reply:
I kept the `:empty:before` version for now since it both scales with font size changes and it let me remove more quirky bits of `.bocTextValue.readOnly .content span[tabindex]:empty` as well as having it properly sized and aligned without properties like `min-height: 1.5em; padding-top: 0; padding-bottom: 0;` (while some of them would still be needed for v-align). It's not necessary anymore to set these so that the size and complexity of the CSS is further reduced.